src/lib/Scenes/Collection/Components/FeaturedArtists.tsx

```diff
@@ -123,21 +105,11 @@ export class FeaturedArtists extends React.Component<FeaturedArtistsProps, Featu
 
 export const CollectionFeaturedArtistsContainer = createFragmentContainer(FeaturedArtists, {
   collection: graphql`
-    fragment FeaturedArtists_collection on MarketingCollection
-      @argumentDefinitions(screenWidth: { type: "Int", defaultValue: 500 }) {
+    fragment FeaturedArtists_collection on MarketingCollection {
       artworksConnection(aggregations: [MERCHANDISABLE_ARTISTS], size: 0, sort: "-decayed_merch") {
         merchandisableArtists(size: 9) {
-          slug
           internalID
-          name
-          image {
-            resized(width: $screenWidth) {
```

Comment:
The `ArtistListItem` component uses the original-sized image. A follow-up PR could change that to use the resized version, like we were originally doing in `FeaturedArtists`.